cl-base.owl

```diff
@@ -210,6 +210,12 @@
     
 
 
+    <!-- http://purl.org/dc/elements/1.1/creator -->
+
+    <owl:AnnotationProperty rdf:about="http://purl.org/dc/elements/1.1/creator"/>
```

Comment:
Where does this come from? We are normally not using dc/elements properties anymore, we are using dc/terms instead.

Reply:
> Not doing so should break QC checks.

It does not because `dc/elements/1.1/creator` has been [specifically added](https://github.com/obophenotype/cell-ontology/pull/3441) to the list of allowed annotation properties.